internal/plugins/ansible/v1/scaffolds/init.go

```diff
@@ -102,7 +108,11 @@ func (s *initScaffolder) scaffold() error {
 		&kdefault.Kustomize{},
 		&kdefault.AuthProxyPatch{},
 
-		&templates.Makefile{},
+		&templates.Makefile{
+			KustomizeVersion:       KustomizeVersion,
```

Comment:
As far as I can tell, `KustomizeVersion` is only used in the Makefile template. If that's the case, I think I would prefer that we remove `KustomizeVersion` from this file and manage it entirely in the Makefile template.

Reply:
I like the idea of having all versions centralized to be used in the templates in the order we are not in the position of having 2 templates with diff values for the same constants. However, please ensure that the change here that final changes here will be reflected in the helm as well.

Reply:
Another idea is to reference `sigs.k8s.io/kubebuilder/pkg/plugin/v2/scaffolds` for `KustomizeVersion`.